steppy/adapter.py

```diff
@@ -14,10 +14,57 @@ class AdapterError(Exception):
 
 
 class Adapter:
+    """Translates outputs from parent steps to inputs to the current step
+
+    Attributes:
+        adapting_recipes: The recipes that the adapter was initialized with
+
+    """
     def __init__(self, adapting_recipes: Dict[str, AdaptingRecipe]):
+        """Adapter constructor
+
+        Note:
+            You have to import the extractor 'E' from this module to construct
+            adapters
+
+        Args:
+            adapting_recipes: Recipes used to control the input translation.
```

Comment:
I would add that `adapting_recipe` may be any literal and `E` is just a placeholder for values extracted from the outputs of previous steps.

Reply:
@grzes314 Ok, please add it.
@grzes314 can you also review all docstrings in `adapter.py` and add Example there? THX